lib/src/components/NxVulnerabilityIndicator/__tests__/NxVulnerabilityIndicator.test.tsx

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import NxVulnerabilityIndicator, { Props } from '../NxVulnerabilityIndicator';
+import { rtlRender, runTimers, userEvent } from '../../../__testutils__/rtlUtils';
+import { screen } from '@testing-library/react';
+
+describe('NxVulnerabilityIndicator', function() {
+  const quickRender = rtlRender<Props>(NxVulnerabilityIndicator, {});
+
+  it('renders an element with a role of graphics-symbol', async function() {
+    const el = quickRender();
+    await runTimers();
+    const vulnerabilityIndicator = el.getByRole('graphics-symbol');
+    expect(vulnerabilityIndicator).toBeInTheDocument();
+  });
+
+  it('has a fallback role of img', async function() {
+    const el = quickRender();
+    await runTimers();
+    const vulnerabilityIndicator = el.getByRole('img', { queryFallbacks: true });
+    expect(vulnerabilityIndicator).toBeInTheDocument();
+  });
+
+  it('takes precedence of severityLevel if both props are provided', async function() {
+    const el = quickRender({ score: '9.0', severityLevel: 'low' });
+    await runTimers();
+    const user = userEvent.setup(),
+        vulnerabilityIndicator = el.getByRole('graphics-symbol')!;
+
+    await user.hover(vulnerabilityIndicator);
+    const tooltip = await screen.findByRole('tooltip');
+    expect(tooltip).toHaveTextContent('Low');
+  });
+
+  it('sets the accessible name based on the severity level', async function() {
+    const el = quickRender({ score: '9.0', severityLevel: 'low' });
+    await runTimers();
+    const vulnerabilityIndicator = el.getByRole('graphics-symbol')!;
+
+    expect(vulnerabilityIndicator).toHaveAccessibleName('Low');
+  });
+
+  describe('should have default tooltips', function() {
+    const getTooltipTextForProps = async (vulnerability: Props) => {
+      const el = quickRender(vulnerability)!;
+      await runTimers();
+      const user = userEvent.setup(),
+          vulnerabilityIndicator = el.getByRole('graphics-symbol');
+
+      await user.hover(vulnerabilityIndicator);
+      const tooltip = await screen.findByRole('tooltip');
+      return tooltip.textContent;
+    };
+
+    describe('when vulnerabilityScore prop is provided', function() {
```

Comment:
Fewer tests of values in the middle of each range, more tests of values at the edges of each range.

Reply:
> Fewer tests of values in the middle of each range, more tests of values at the edges of each range.

Updated test: [7d9d07f](https://github.com/sonatype/sonatype-react-shared-components/pull/1199/commits/7d9d07f4f44bfdee3e1d62a911acc76f4df13d1b)